src/EnergyPlus/DataSizing.hh

```diff
@@ -1236,7 +1248,7 @@ struct SizingData : BaseGlobalStruct
         this->CurZoneEqNum = 0;
```

Comment:
I think you can just say `*this = SizingData();` or maybe (`new(this) SizingData();` would be better because it avoids the copy) for the body of `clear_state` because all of the defaults are in the declaration. See examples of this throughout the code, e.g., ZonePlenum.hh